### PR TITLE
feat: premium feed admin

### LINKED ISSUE #939: premium feed admin

### PR DESCRIPTION
**Pull request by ChiragPansuriya-iView**
Premium Admin Setup design 
Screenshot 2024-02-08 at 4 40 12 PM
Screenshot 2024-02-08 at 4 40 33 PM
Closes #939

### CODE REVIEW

**Comment by ChiragPansuriya-iView** [comment]
FYI ,
Tier name looks like that in Figma, but I made different. I am working on matching the Figma.
Screenshot 2024-02-08 at 5 22 54 PM

File: packages/screens/UserPublicProfile/components/accordion/AccordionComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+      }}
+    >
+      <TouchableOpacity
+        style={{ flexDirection: "row", justifyContent: "space-between" }}
+        onPress={() => {
+          setIsopen(!isopen);
+        }}
+      >
+        <View style={{ flexDirection: "row", alignItems: "center" }}>
+          <Image
```
Comment (clegirar) [inline packages/screens/UserPublicProfile/components/accordion/AccordionComponent.tsx:74]: Please use `OptimizedImage` instead 👍

File: packages/components/hub/ProfileButton.tsx [hunk 2]
```diff
@@ -71,7 +71,7 @@ export const ProfileButton: React.FC<{
         }
       >
         <SecondaryButtonOutline
-          size="XL"
+          size="M"
```
Comment (clegirar) [inline packages/components/hub/ProfileButton.tsx:74]: This change didn't create any regression on other components that use it ?
If you don't know please take care of that 👍

**Review by clegirar — CHANGES_REQUESTED** [review]
It's a really good practice to add specific `TextInputs`, really like it 👍 BUT, we have to minimize the number of variants of it 👍 Can you check if other components already used can be fit with your behavior (sometime it can be a design issue) ?

Second point found some components/files a little too long, `SubscriptionSetupModal.tsx`, `AccordionComponent.tsx` and the `.web.tsx` file. Can you do something to make it more readable 👍

**Comment by clegirar** [comment]
Screenshot 2024-02-08 at 14 33 22
Screenshot 2024-02-08 at 14 33 50

The first one, the text is on 2 lines, and we want it in 1 line 👍 (relating to the figma)
The second one is just to reask you about that https://github.com/TERITORI/teritori-dapp/pull/937#issuecomment-1933960897

**Comment by clegirar** [comment]
For UI stuff:
image

The text and the arrow are not correctly aligned, and i think the start is a little too on top

And the premium sub text is not on one line again

**Pull request merged**